js/tidex2.js

```diff
@@ -0,0 +1,960 @@
+        //     {
+        //         "at":1646289676,
+        //         "ticker":{
+        //             "name":"yfi_usdt",
+        //             "bid":"21937.47155034",
+        //             "ask":"21966.13380147",
+        //             "open":"21449.08",
+        //             "high":"22619.38",
+        //             "low":"21193.59",
+        //             "last":"21968.03",
+        //             "vol":"188.4452385",
+        //             "deal":"4075398.065557751",
+        //             "change":"2"
+        //         }
+        //     }
+        //
+        const timestamp = this.safeTimestamp (ticker, 'at');
+        if (timestamp !== undefined) {
+            ticker = this.safeValue (ticker, 'ticker');
+        }
+        const marketId = this.safeStringUpper (ticker, 'name');
+        const symbol = this.safeSymbol (marketId, market);
+        const last = this.safeString (ticker, 'last');
+        return this.safeTicker ({
+            'symbol': symbol,
+            'timestamp': timestamp,
+            'datetime': this.iso8601 (timestamp),
+            'high': this.safeString (ticker, 'high'),
+            'low': this.safeString (ticker, 'low'),
+            'bid': this.safeString (ticker, 'buy'),
+            'bidVolume': undefined,
+            'ask': this.safeString (ticker, 'sell'),
+            'askVolume': undefined,
+            'vwap': undefined,
+            'open': undefined,
+            'close': last,
+            'last': last,
+            'previousClose': undefined,
+            'change': undefined,
+            'percentage': undefined,
+            'average': this.safeString (ticker, 'avg'),
+            'baseVolume': this.safeString (ticker, 'vol_cur'),
+            'quoteVolume': this.safeString (ticker, 'vol'),
+            'info': ticker,
+        }, market, false);
+    }
+
+    async fetchTickers (symbols = undefined, params = {}) {
+        await this.loadMarkets ();
+        const response = await this.publicGetTickers (params);
+        const result = this.safeValue (response, 'result');
+        //
+        //     {
+        //         "code":200,
+        //         "success":true,
+        //         "message":"",
+        //         "result":{
+        //             "YFI_USDT":{
+        //                 "at":1646289676,
+        //                 "ticker":{
+        //                     "name":"yfi_usdt",
+        //                     "bid":"21937.47155034",
+        //                     "ask":"21966.13380147",
+        //                     "open":"21449.08",
+        //                     "high":"22619.38",
+        //                     "low":"21193.59",
+        //                     "last":"21968.03",
+        //                     "vol":"188.4452385",
+        //                     "deal":"4075398.065557751",
+        //                     "change":"2"
+        //                 }
+        //             }
+        //         }
+        //     }
+        //
```

Comment:
also I would add an extra line here to extract the tickers as a list from the `result` object since we take the symbol from the `name` field in `parseTicker ()` anyway, just to be safe like so:

```JavaScript
const result = this.safeValue (response, 'result', {});
const tickers = Object.values (result);
return this.parseTickers (result, symbols);
```

(I did test and it does parseTicker seems to handle key-value objects in php and python but I'm thinking just to be extra safe) though double check this with @kroitor and @frosty00